ext/fileinfo/libmagic/magic.c

```diff
@@ -403,3 +403,12 @@ magic_getparam(struct magic_set *ms, int param, void *val)
 		return -1;
 	}
 }
+
+public const char *
+magic_magic_file_get(struct magic_set *ms)
+{
+	if(strncmp(ms->file, "unknown", strlen("unknown")) == 0) {
```

Comment:
`sizeof("unknown")-1` please. But otherwise, don't you think it would make sense to indicate the built-in database is used?